src/sbm_gwf_model.jl

```diff
@@ -511,10 +511,13 @@ function update_sbm_gwf(model)
     # update groundwater domain
     update(lateral.subsurface.flow, Q, Δt_sbm)
 
-    # determine excess head [m] (exfiltwater) in groundwater domain (head > surface) and reset head
+    # determine excess water depth [m] (exfiltwater) in groundwater domain (head > surface)
+    # and reset head
     exfiltwater =
-        lateral.subsurface.flow.aquifer.head .-
-        min.(lateral.subsurface.flow.aquifer.head, lateral.subsurface.flow.aquifer.top)
+        (
+            lateral.subsurface.flow.aquifer.head .-
+            min.(lateral.subsurface.flow.aquifer.head, lateral.subsurface.flow.aquifer.top)
+        ) .* lateral.subsurface.flow.aquifer.specific_yield
```

Comment:
It might be worthwhile to replace `lateral.subsurface.flow.aquifer.specific_yield` by `storativity(lateral.subsurface.flow.aquifer)`, since the `storativity` function will pick the right term also for a confined aquifer. 

For the record, this is the storativity function defined in aquifer.jl:
```
storativity(A::UnconfinedAquifer) = A.specific_yield
storativity(A::ConfinedAquifer) = A.storativity
```

The sbm_gwf model now always uses the unconfined aquifer and so specific yield is currently always the right one, but should somebody e.g. experiment with an sbm_gwf with confined, this will ensure it still works. And there's no real downsides.

Reply:
Yes, I agree, see commit a49fc6d.